src/main/resources/application-local.properties

```diff
@@ -2,4 +2,17 @@ spring.datasource.url=jdbc:h2:mem:demo;DB_CLOSE_ON_EXIT=FALSE
 spring.datasource.username=sa
 spring.datasource.password=
 spring.datasource.driver-class-name=org.h2.Driver
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
\ No newline at end of file
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+## ??? ???????????? ????? ????????? ? Keycloak ?????? ? ????????????? ??????? ???????
+## ???? ??? ???????? ????? ??????????? ?? keycloak.
+keycloak.auth-server-url = http://127.0.0.1:8080/realms/SpringBootKeycloak
+keycloak.realm = demorealm
+keycloak.ssl-required = external
+keycloak.resource = demoapp
+keycloak.credentials.secret = 11111111-1111-1111-1111-111111111111
+keycloak.use-resource-role-mappings = true
+keycloak.clientName="admin"
+keycloak.client-key-password="admin"
+server.port = 8080
```

Comment:
Why do we have it here? This is a default server port anyway

Reply:
```
keycloak.server-url
keycloak.realm
keycloak.client-id
keycloak.client-secret
```

Please - have this properties here only. And map them in configuration properties.